src/main/java/ca/corefacility/bioinformatics/irida/ria/web/projects/settings/ProjectSettingsRemoteController.java

```diff
@@ -132,4 +137,20 @@ public String getProjectSettingsRemotePage(@PathVariable Long projectId, final M
 		projectControllerUtils.getProjectTemplateDetails(model, principal, project);
 		return "projects/settings/pages/remote";
 	}
+
+
+	@RequestMapping("/remote-settings")
```

Comment:
Comments

Reply:
Added in 68d3acaeba753d0395ebb82cf7b475f5af7d89c4